ui/packages/catalog-ui-search/src/main/webapp/component/singletons/metacard-definitions.js

```diff
@@ -222,6 +261,15 @@ module.exports = new (Backbone.Model.extend({
   getMetacardStartingTypes() {
     return metacardStartingTypes
   },
+  getCountries(type) {
+    $.get('./internal/enumerations/countries', {
```

Comment:
❓ Any reason to use the jQuery get vs the native fetch?

Reply:
A poor but honest reasoning -- there was already a dependency & usage for jQuery in this file and thought that's the standard for our non-react UI code..